src/applications/hca/config/form.js

```diff
@@ -508,7 +510,12 @@ const formConfig = {
           title: 'Service history',
           uiSchema: {
             'ui:title': 'Service history',
-            'ui:description': 'Check all that apply to you.',
+            'ui:description': !environment.isProduction()
+              ? MilitaryPrefillMessage
+              : undefined,
+            'view:textObject': {
```

Comment:
@jbalboni , for adding prefill-message to pages that already have 'ui:description' prop, unlike the 2 solution-ideas we discussed yesterday, I found a much easier, cleaner way -- 'view:textObject' prop.